Makefile.pre.in

```diff
@@ -434,6 +434,7 @@ PYTHON_OBJS=	\
 		Python/context.o \
 		Python/critical_section.o \
 		Python/crossinterp.o \
+		Python/cpuinfo.o \
```

Comment:
Maybe it should be moved to the `context.o`.

（co, cp, cr, ....)

Reply:
Oh right, thanks!